server/src/sim/plotccocs.py

```diff
@@ -1,16 +1,23 @@
 """
 Plots cost-coverage, coverage-outcome and cost-outcome curves
-    
+
 Version: 2015jan19 by robynstuart
 """
 from matplotlib.pylab import figure, plot, hold, xlabel, ylabel, title, xlim, ylim
 from printv import printv
 
+# TODO robyn - which one is the correct one? I had a merge conflict
+# default_progname = 'ART'
+# default_ccparams = [2.0, 0.2, 2500000.0, None, None] #
+# default_ccplot = [1e4, None, 1]
+# default_coparams = [0.3, 0.5, 0.7, 0.9]
+
```

Comment:
@robynstuart there was a merge conflict with the default data. I took the one from develop, but wasn't sure which one is actually the correct one. Can you check this?